orchestra/templates/orchestra/base.html

```diff
@@ -78,13 +75,11 @@
     {% endblock %}
 
     <!-- js placed at the end of the document so the pages load faster -->
-    {% compress js %}
-    <script src="{% static "orchestra/common/js/lib/jquery.js" %}"></script>
+    <script src="{% static "dist/orchestra/common/js/lib/jquery.min.js" %}"></script>

```

Comment:
i don't think we should include any libs inside of the package, but we should move that do a different PR